<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseExtensions</class>
 <widget class="QMainWindow" name="ChooseExtensions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set Rules</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fbfbfc;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1a1a1d;
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>SMARTIFY FILE REMOVER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step 2 of 3</string>
    </property>
   </widget>
   <widget class="QListWidget" name="availableExtensions">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000;

color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000000;

color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QListWidget" name="choosedExtensions">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose the extensions you want to delete in the below folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Available Extensions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Choosen Extensions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="putInToChoosedExtensions">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromChoosedExtensions">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;</string>
    </property>
    <property name="text">
     <string>/home</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
